### ISSUE TITLE
Default values for QoS for Daemonset are too aggressive.

### ISSUE THREAD

**Issue by dbayendor-cb** [wavefrontHQ/helm#2]
In testing the helm chart on my production and test clusters  I have found that the published default chart values are set for Guaranteed QoS class, which prevents the daemonset from launching pods on heavily loaded nodes with stable workloads.  What I am seeing is that typically only 60% of my worker nodes can schedule the wavefront collector, the rest remain in `pending`.

The published chart values are set for Guaranteed QoS class, and are very aggressive, at least 10 times higher limits than I have seen in real-world use.  

The daemonset deploys properly and reliably when BestEffort QoS is configured, which is in line with other daemon sets on my cluster (kube2iam, fluent-bit, etc).

The override in the `values.yaml` I am currently using looks like this:
```
## Override the default resource requests to set BestEffort QoS class
## Published chart values are set for Guaranteed QoS class, which prevents
## daemonset from launching pods on heavily loaded nodes
collector:
  resources:
    requests:
      cpu:
      memory:
    limits:
      cpu:
      memory:
```

**Comment by dbayendor-cb** [comment]
Any feedback on this request?

**Comment by vikramraman** [comment]
@dbayendor-cb It was addressed as part of this commit: https://github.com/wavefrontHQ/helm/commit/fef859c8b2b250b4b136bc94788b6a1d6485f176 and released with v1.1.4 of the wavefront chart.

**Comment by dbayendor-cb** [comment]
Thanks.
